leiden-journal-of-international-law.csl

```diff
@@ -0,0 +1,637 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="en-GB" and="symbol" et-al-min="3" initialize-with="." sort-separator=" " initialize-with-hyphen="false" demote-non-dropping-particle="never">
+  <info>
+    <title>Leiden Journal of International Law</title>
+    <title-short>LJIL</title-short>
+    <id>"http://www.zotero.org/styles/leiden-journal-of-international-law.csl</id>
+    <link href="http://www.zotero.org/styles//leiden-journal-of-international-law.csl" rel="self"/>
```

Comment:
`href="http://www.zotero.org/styles//leiden-journal-of-international-law.csl"` should be `href="http://www.zotero.org/styles/leiden-journal-of-international-law"`